avm-transpiler/src/transpile.rs

```diff
@@ -367,12 +519,7 @@ fn map_brillig_pcs_to_avm_pcs(initial_offset: usize, brillig: &Brillig) -> Vec<u
     pc_map[0] = initial_offset;
     for i in 0..brillig.bytecode.len() - 1 {
         let num_avm_instrs_for_this_brillig_instr = match &brillig.bytecode[i] {
-            BrilligOpcode::Load { .. } => 2,
-            BrilligOpcode::Store { .. } => 2,
```

Comment:
Was this a bug where these were left as 2 :eyes: oof that would've been a time killer debugging that

Reply:
probably took an hour to find lmao, the avm got stuck in a loop and i couldnt work out why it was reverting